spec/models/item_spec.rb

```diff
@@ -74,4 +74,11 @@
     create(:item_status, id: 1)
     expect(item.item_status.state).to eq("Completed")
   end
+
+  it "has an associated business" do
+    company = create(:business)
+    item.business = Business.find(company.id)
+    item.save
```

Comment:
You don't have to have this line in the test I don't think. It's fine either way.